.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.yml

```diff
@@ -0,0 +1,60 @@
+name: 🐛 Bug Report
+description: Report an unexpexted behaviour
+title: "[Bug]: "
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Issues are formatted using markdown, a useful summary of the features provided by GitHub can be found [here](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf).
+  - type: textarea
+    attributes:
+      label: Expected behaviour ?
+      description: Please describe the behavior you are expecting
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Please provide detailed steps for reproducing the issue.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
```

Comment:
There are two expected behavior. One should be enough

Reply:
Oops, it's a mistake.